src/examples/surgicalplanning/alternative.js

```diff
@@ -0,0 +1,22 @@
+import Sandcastle from "sandcastle"
+import { Vector3 } from "three"
+const GLTFHeart = require("./assets/models/heart/heart.glb");
+
+Sandcastle.run(async (scene, controllers) => {
```

Comment:
An alternative, on-rails version of SandCastle. Probably too far on the "easy of use" side at the expense of flexibility, but just wanted to throw some ideas out there.

Reply:
To restate my sentiment: YES, I'd love to see that component!:). Also a super quick way to XRPackagify sandcastle "model viewers", although does present the question of when a simple `gltf XRPackage` (a future, separate XRPackage format) would be more appropriate.